modules/jetty-http-server/src/main/java/com/spotify/apollo/http/server/JettyHttpRequestMetadata.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * -\-\-
+ * Spotify Apollo Jetty HTTP Server Module
+ * --
+ * Copyright (C) 2013 - 2016 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+package com.spotify.apollo.http.server;
+
+/**
+ * Defines the metadata reported for {@link com.spotify.apollo.RequestContext}s originating from
+ * this server.
+ */
+public enum JettyHttpRequestMetadata {
```

Comment:
The purpose of this enum is to document what metadata gets added by the Jetty HTTP server. The client can iterate through the contents of the metadata map, so the data can be read without knowing about this enum.